stdlib/public/SwiftShims/CMakeLists.txt

```diff
@@ -106,34 +84,47 @@ function(find_llvm_clang_headers suffix description out_var)
 endfunction()
 
 if(SWIFT_BUILT_STANDALONE)
-  find_llvm_clang_headers("/clang/${CLANG_VERSION}" "Clang"
-    clang_headers_location)
+  find_llvm_clang_headers("/clang/${CLANG_VERSION}" "Clang" clang_headers_location)
 else() # NOT SWIFT_BUILT_STANDALONE
   set(clang_headers_location "${LLVM_LIBRARY_DIR}/clang/${CLANG_VERSION}")
 endif()
 
 if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
-  set(cmake_symlink_option "copy_directory")
+  set(cmake_symlink_option copy_directory)
```

Comment:
A note:

We should be consistent about this. Later on in the PR, (originally) compnerd changed symlink -> copy. From talking with compnerd, he said that these cases where we were conditional was due to us needing to audit the code here. Later on (I am going to add a comment) where he switched from using symlink -> copy, he is going to switch back to symlink and then audit/change things in a subsequent commit.